### ISSUE TITLE
aria-selected on option element should flag a warning/needs review

### ISSUE THREAD

**Issue by carmacleod** [IBMa/equal-access#344]
Steps:
- go to https://w3c.github.io/html-aria/tests/option-aria-select.html
- run accessibility checker
- note that there are no warnings (or violations) flagged for use of aria-selected on option element

Spec says: https://w3c.github.io/html-aria/#el-input-option
> Developers SHOULD NOT use the aria-selected attribute on the option element.

Screen snapshot of `option aria-selected="true"` in Test 1, showing no violations, warnings, or info. This should be a "needs review".

![image](https://user-images.githubusercontent.com/3331913/110557706-1d3d9600-810f-11eb-8c3b-83f8114afe40.png)

**Comment by carmacleod** [comment]
Spoke to @scottaohara about whether it would be ok to flag a violation instead of a warning, and he says that would be ok.
The basic point is to tell authors not to do it.

**Comment by MHoov** [comment]
As of June 1
- [x] general rule description (SME & Dev)
- [x] testcases written (Dev)
- [ ] rule developed  (Dev)
- [ ] message written (SME)
- [ ] help written (SME)

**Comment by marcjohlic** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Equal-Access-61ae71f9f8b4ec001e136462/issues/IBMa/equal-access/344?planning-poker) with ZenHub @aliunwala @mbgower @shunguoy @tombrunet @philljenkins @drjoho

**Comment by shunguoy** [comment]
close this issue as this is covered in #721 , and the datalist element issue is covered in #729.
